share/VERSION.txt

```diff
@@ -1 +1 @@
-1.9.0-SNAPSHOT
```

Comment:
I think, this is not related to this change. Shouldn't it be updated automatically after a release? We might create a JIRA for it...

Reply:
Indeed it is not related to this issue, however without this change Python tests failed, that's why I added it to the PR.

Reply:
Resolving this comment, as the problem is addressed in #527